website/pages/en/versions.js

```diff
@@ -16,36 +16,72 @@ const CWD = process.cwd();
 
 const versions = require(`${CWD}/versions.json`);
 
-// paths
-const PDFPath = `${CWD}/static/download/download-pdf.json`;
-const windowsPath = `${CWD}/static/download/download-windows.json`;
-const linuxPath = `${CWD}/static/download/download-linux.json`;
-const macPath = `${CWD}/static/download/download-mac.json`;
+// downloads
+const downloadJSONPath = `${CWD}/static/download/download.json`;
```

Comment:
How come we don't just call the github api directly from the page? Not quite clear how this will work if the build is ran in a release?

Reply:
The download.json is a parsed version which has all bits we want in it, but could do it at run time. For now I made it run at build with f520492, then for the release it waits for the PDF to finish building and then it makes the versions page again

Reply:
What will this do on systems without an internet connection? i.e. consoles?

I thought it would just end up not showing because it wouldn't be able to reach the github api but I think it will show up now?

Reply:
Ah yeah... I think 6449974 will resolve this. It won't have the download options in any of the builds, then after the release has finished it builds the downloads page with the latest assets.

_As the pages are built to HTML, not React pages, calling the GitHub API directly from the page using JS would mean the content wasn't static which wouldn't be great for SEO IMO_

Reply:
Ah yeah I hadn't really considered the SEO aspect of it. Sounds like this shouldn't show up in the app then. Lets see!